### PR TITLE
Mde/simulator05x

### PR DESCRIPTION
**Pull request by dmarienko**
Provides ctx.exchanges property in StrategyContext

### CODE REVIEW

File: src/qubx/core/mixins/trading.py [hunk 1]
```diff
@@ -82,3 +82,6 @@ def _generate_order_client_id(self, symbol: str) -> str:
         assert self._order_id is not None
         self._order_id += 1
         return "_".join(["qubx", symbol, str(self._order_id)])
+
+    def exchanges(self) -> list[str]:
```
Comment (yuriy-arabskyy) [inline src/qubx/core/mixins/trading.py:86]: You envision the trading manager as the router between different brokers? So the multi-exchange logic would go into this class.

Comment (dmarienko) [reply src/qubx/core/mixins/trading.py:86]: Yes, correct

File: src/qubx/core/interfaces.py [hunk 2]
```diff
@@ -877,6 +891,13 @@ def is_simulation(self) -> bool:
         """
         ...
 
+    @property
+    def exchanges(self) -> list[str]:
```
[the remark this answers is not in the document]
Comment (dmarienko) [reply src/qubx/core/interfaces.py:895]: Yep, that's still a pain in the ass )))
Flexibility leads to redundancy. From another point of view - it's quite rare case if we read data from one source but trade on another exchange. So I propose to use market data's exchange for now

[the remark this answers is not in the document]
Comment (yuriy-arabskyy) [reply src/qubx/core/interfaces.py:895]: yes, I agree. Let's only add once actually needed and keep it simple.

**Review by yuriy-arabskyy — APPROVED** [review]

**Pull request merged**